src/controllers/list.js

```diff
@@ -962,10 +962,7 @@ class ItemsView {
 
     getSortValues() {
         const basekey = this.getSettingsKey();
-        return {
-            sortBy: userSettings.getFilter(basekey + '-sortby') || this.getDefaultSortBy(),
```

Comment:
We need `this.getDefaultSortBy()`. Maybe pass it as `defaultSortBy` argument?

Reply:
Going to implement your suggestions. Thank you once again :pray: